app/models/client.rb

```diff
@@ -88,7 +88,7 @@ def self.sortable_intake_attributes
   end
 
   delegate *delegated_intake_attributes, to: :intake
-  scope :after_consent, -> { distinct.joins(:tax_returns).merge(TaxReturn.where.not(status: "intake_before_consent")) }
+  scope :after_consent, -> { joins(:intake).where("intakes.primary_consented_to_service_at IS NOT NULL") }
```

Comment:
<img width="1091" alt="Screen Shot 2022-04-11 at 12 56 26 PM" src="https://user-images.githubusercontent.com/4494389/162826869-cae928d7-197b-4a5d-89ce-14758d70f3a6.png">

<img width="780" alt="Screen Shot 2022-04-11 at 12 56 52 PM" src="https://user-images.githubusercontent.com/4494389/162826891-9780c354-2c3a-4023-b14f-0de613c31bee.png">

I got this SQL error when i had it written the more Railsy way. I THINK the search gem we use was interfering with the joins on intake?